### DISCUSSION TITLE
blog/01-getting-started

### DISCUSSION THREAD

**Discussion by giscus** [trevortylerlee/astro-micro#6] [Announcements]
# blog/01-getting-started

Hit the ground running.

https://blogsite-mhrg33i5c-kenricqqgmailcoms-projects.vercel.app/blog/01-getting-started

**Comment by saaryul** [comment]
well done

**Reply by saaryul** [reply]
wow done

**Comment by alirezax2** [comment]
looks good

**Comment by linuxfandudeguy** [comment]
woah
